modules/ROOT/pages/import-api-def-dc.adoc

```diff
@@ -1,35 +1,39 @@
-= To Import an API Definition from Design Center
+= Importing an API Definition from Design Center
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 Anypoint Studio allows you to import an API Definition from Design Center and implement it either into a new Mule Project, or an existing one.
 
-== To Implement an API imported from Design Center Into a New Project
+== Implementing an API imported from Design Center Into a New Project
 
-. Click File > New > Mule Project to create a project, and set the field values in the project wizard:
+. Select *File* > *New* > *Mule Project* to create a project, and set the field values in the project wizard:
```

Comment:
I would split this step into two steps:

1. Select *File* > *New* > *Mule Project* to create a project.
2. Set the field values in the project wizard: